.github/workflows/circular.yml

```diff
@@ -32,6 +32,9 @@ jobs:
         run: sudo apt-get install -y libgraphviz-dev
       - name: Install dependencies
         run: python -m pip install tox
+      - name: Initialize tox
+        run: |
+          tox -e init
```

Comment:
If we merge this as it is all PRs from repositories that do not have an `init` environment will be broken, as all GHA jobs will fail (if the `tox` environment is not known it gives an error)

Is `tox -e init || true` valid syntax in GHA? 🤔

Reply:
New commit should fix that (I first check for init being in the list of environments)